src/utils/fetchData.js

```diff
@@ -1,19 +1,21 @@
 import queryPerfData from '@mozilla-frontend-infra/perf-goggles';
-import { BENCHMARKS, CONFIG } from '../config';
+import linux64 from '../configuration/js-team/linux64';
 import prepareData from './prepareData';
 import { convertToSeconds } from './timeRangeUtils';
+import DASHBOARD_CONFIG from '../configuration/appDefaults';
+import platforms from '../configuration/js-team/index';
 
 const fetchData = async (platform, benchmark, range) => {
   const ALL_DATA = {};
 
-  const fetchIt = async (configUID, overview = false, timeRange = CONFIG.default.timeRange) => {
+  const fetchIt = async (configUID, overview = false, timeRange = DASHBOARD_CONFIG.timeRange) => {
     const includeSubtests = !overview;
-    const comparingBenchmarks = Object.keys(BENCHMARKS[configUID].compare);
+    const comparingBenchmarks = Object.keys(linux64.benchmarks[configUID].compare);
     return Promise.all(comparingBenchmarks
       .map(async (modeKey) => {
-        const benchmarkOptions = BENCHMARKS[configUID].compare[modeKey];
+        const benchmarkOptions = linux64.benchmarks[configUID].compare[modeKey];
```

Comment:
Hi @armenzg, I need your thoughts here. 
`BENCHMARKS` was included from the old config file which had all benchmarks. We have now separated all benchmarks in their platform configs, so I got all benchmarks through `linux64.benchmarks`  because Linux has all benchmarks in it.

2nd option is to keep all benchmarks in a separate file as in old config but that would be unnecessary repetition. 
Also, in the first solution, I am not sure what changes will occur in the Linux platform in the future.

Reply:
This is where it gets tricky.

You should be able to refactor `fetchData(platform, benchmark, range)` to something like `fetchData(benchmarks, range)`. At the end of the day, what this function does it built a `seriesConfig` that can be passed to `queryPerfData`.

Get a hold of me on IRC if you can't figure out after spending 30 to 60 minutes.

Reply:
```
 const seriesConfig = {
          platform: platforms[platform].platform,
          option: benchmarkOptions.buildType,
          ...benchmarkOptions,
        };
```
This seriesConfigs consists of `platform` and `option` keys which depends on platfrom and benchmarks. If we do not pass `platform` how would we make it? 

Are we allowed to do anything out of the function or inside function? 
We need all `Benchmarks` and `platform` here.

Reply:
I think if you take a step back and try to solve [this issue](https://github.com/mozilla-frontend-infra/firefox-performance-dashboard/issues/97) instead you will have an easier time on this issue.

There's code changes that have to happen to this function OR even config structure changes. At this point, I can only give you more specific answers if I actually tried to fix the issue.

If you need to I can fix issue 97 and this PR would be easier. I can give you another issue meanwhile.

Reply:
I have no issues, I can do whatever you suggest better for me. 
I can give a shot to issue 97 it think it will help me more to understand the things & flow. 

You tell what should I do?

Reply:
Tackle issue 97 and we can comeback to this one later.
That issue should give you a better understanding of the whole project as a whole and will knock off some work in here.